chapter_01/exercises/EXO_1.8_type_in_program_test.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2019 Cheick Bamba
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+from math import pi
+h = 5 # height
+b = 2 # base
+r = 1.5 # radius
+area_parallelogram = h*b
+print (' The area of the parallelogram is %.3f' % (area_parallelogram))
```

Comment:
You can verify your results by comparing them with an expected value. You can use Python *assertion* for this:
https://www.tutorialspoint.com/python/assertions_in_python.htm